src/docfx/validation/ContentValidator.cs

```diff
@@ -15,7 +15,8 @@ namespace Microsoft.Docs.Build
     internal class ContentValidator
     {
         // Now Docs.Validation only support conceptual page, redirection page and toc file. Other type will be supported later.
-        private static readonly string[] s_supportedPageTypes = { "conceptual", "includes", "toc", "redirection" };
+        // Learn content: "learningpath", "module", "moduleunit"
+        private static readonly string[] s_supportedPageTypes = { "conceptual", "includes", "toc", "redirection", "learningpath", "module", "moduleunit" };
```

Comment:
Update the page type mapping to include "learn", base on `docs-ui`, these are the `learn` schemas:
![image](https://user-images.githubusercontent.com/511355/93297756-801b3800-f824-11ea-9921-1a937394d6b7.png)


```suggestion
        private static readonly string[] s_supportedPageTypes = { "conceptual", "includes", "toc", "redirection", "learn" };
```

Reply:
If we use `Learn` as page type, then in below dialog we will have to merge all `Learn.*` entries into only one entry `Learn`? @OsmondJiang 
![image](https://user-images.githubusercontent.com/226552/93300298-19e4e400-f829-11ea-9a79-e6fe6224aada.png)

Reply:
Megan require to use Page kind instead of Page type